### ISSUE TITLE
Selected project is not saved

### ISSUE THREAD

**Issue by Neustradamus** [Flyspray/flyspray#835]
Dear @flyspray team, @peterdd,

We are here, we are in a project:
- https://domain.tld/

For example, we select "All projects", we obtain this link in browser: https://domain.tld/index.php?project=0&do=index&switch=1

After, we type https://domain.tld/ and we are not in "All projects" but in a project.

Note: It works with all another projects, the homepage is always the same project.

Can you solve it?

**Comment by Neustradamus** [comment]
@peterdd: Yes about settings from host but if an user selects a specified project, it must be the selected project in the current session...

**Comment by peterdd** [comment]
I made the explicit decision to move that from session to url to enable working with multiple tabs in browser.

If it is a session variable handling multiple tabs with multiple projects leads to more confusion.

**Comment by Neustradamus** [comment]
@peterdd: An user option can be added to choice if it must be saved or not?

**Comment by Neustradamus** [comment]
@peterdd: Have you seen my latest comment?

**Comment by Neustradamus** [comment]
@peterdd: I have recently talked with several users, and it is needed to solve this problem.

I think that it is possible to add a new option (not enabled by default) to do not save the current selected project used.
